test/e2e/storage/testsuites/stress.go

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This suite tests volumes under stress conditions
+
+package testsuites
+
+import (
+	"context"
+	"sync"
+
+	"github.com/onsi/ginkgo"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	errors "k8s.io/apimachinery/pkg/util/errors"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	"k8s.io/kubernetes/test/e2e/storage/testpatterns"
+)
+
+type stressTestSuite struct {
+	tsInfo TestSuiteInfo
+}
+
+type stressTest struct {
+	config        *PerTestConfig
+	driverCleanup func()
+
+	intreeOps   opCounts
+	migratedOps opCounts
+
+	resources []*VolumeResource
+	pods      []*v1.Pod
+	// stop and wait for any async routines
+	wg      sync.WaitGroup
+	stopChs []chan struct{}
+}
+
+var _ TestSuite = &stressTestSuite{}
+
+// InitStressTestSuite returns stressTestSuite that implements TestSuite interface
+func InitStressTestSuite() TestSuite {
+	return &stressTestSuite{
+		tsInfo: TestSuiteInfo{
+			Name:       "stress",
+			FeatureTag: "[Feature: VolumeStress]",
```

Comment:
This will make the test to be skipped by default. Is it what you want?

Reply:
Yes for now, I plan to create a separate job for this. Right now serial job is already too slow, and this test case is very slow. I also potentially want to tune it a little more and potentially increase parallelism in the future.

Reply:
It's going to be skipped in our e2e (for now), but it will be enabled in e2e/external tests without a possibility to skip it in test manifest. That's odd.